core/Site.php

```diff
@@ -636,4 +636,15 @@ public static function getExcludedQueryParametersFor($idsite)
     {
         return self::getFor($idsite, 'excluded_parameters');
     }
+
+    /**
+     * Returns the user that created this site.
+     *
+     * @param int $idsite The site ID.
+     * @return string|null If null, the site was created before the creation user was tracked.
+     */
+    public static function getCreationUserFor($idsite)
```

Comment:
Maybe `getCreatorLoginFor`? For consistency be also good to add a `public function getCreatorLogin`